backend/src/main/java/ch/puzzle/okr/dto/checkIn/CheckInDto.java

```diff
@@ -0,0 +1,103 @@
+package ch.puzzle.okr.dto.checkIn;
```

Comment:
Wieso verwendest du mit den CheckIn DTO's keine java Records, wie bei den anderen DTO's? 
Besprich dich mal mit Lias.

Reply:
Ich habe hier mit Java Klassen gearbeitet, da die Records in meinem Fall keine Verwendung finden. Lias muss mehrere KeyResultDtos vom Backend zurückgeben und regelt das via Interface welches er von den anderen DTOs implementieren lässt, das bedeutete aber das wir keine Vererbung verwenden können.

Mit Java Klassen können wir uns das Konzept der Vererbung zu nutzen machen und so 3 Klassen (1 abstrakte Parentklasse und 2 Childklassen) anlegen und die gesamten Grundattribute nur im Parent angeben.